x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/fleet_trusted_apps_card.tsx

```diff
@@ -16,19 +16,32 @@ import {
 } from '../../../../../../../../../fleet/public';
 import { useKibana } from '../../../../../../../../../../../src/plugins/kibana_react/public';
 import { getTrustedAppsListPath } from '../../../../../../common/routing';
-import { TrustedAppsListPageRouteState } from '../../../../../../../../common/endpoint/types';
+import {
+  TrustedAppsListPageRouteState,
+  GetExceptionSummaryResponse,
+} from '../../../../../../../../common/endpoint/types';
 import { PLUGIN_ID as FLEET_PLUGIN_ID } from '../../../../../../../../../fleet/common';
 import { MANAGEMENT_APP_ID } from '../../../../../../common/constants';
 import { LinkWithIcon } from './link_with_icon';
-import { TrustedAppItemsSummary } from './trusted_app_items_summary';
+import { ExceptionItemsSummary } from './exception_items_summary';
+import { TrustedAppsHttpService } from '../../../../../trusted_apps/service';
 
 export const FleetTrustedAppsCard = memo<PackageCustomExtensionComponentProps>(({ pkgkey }) => {
   const {
     services: {
       application: { getUrlForApp },
+      http,
     },
-  } = useKibana<{ application: ApplicationStart }>();
+  } = useKibana<CoreStart & { application: ApplicationStart }>();
+
+  const [stats, setStats] = useState<GetExceptionSummaryResponse | undefined>();
+  const [trustedAppsApi] = useState(() => new TrustedAppsHttpService(http));
```

Comment:
Looks like there should be `useMemo` instead of `useState`.

Reply:
Yes, this code was moved here from the exception_items_summary one. I'll do the change